x-pack/plugins/apm/server/lib/transactions/fetcher/index.ts

```diff
@@ -204,7 +202,7 @@ export async function transactionGroupsFetcher(
   };
 }
 
-interface TransactionGroup {
+interface Transaction {
```

Comment:
I think now we are heading into very muddy territory. This is NOT a transaction we've modelled, but a transactionGroup.

My intention was primarily to simplify some folder structures (I think transaction and transaction_group stuff can be in the same folder). But I don't think everything named "transactionGroup" should be named "transaction".